src/app/zap-templates/zcl/zcl.json

```diff
@@ -305,7 +305,37 @@
         ],
         "Temperature Control": ["SupportedTemperatureLevels"],
         "Operational State": ["OperationalState", "OperationalError"],
-        "RVC Operational State": ["OperationalState", "OperationalError"]
+        "RVC Operational State": ["OperationalState", "OperationalError"],
+        "Dishwasher Mode": [
+            "SupportedModes",
```

Comment:
Since the mode instance knows the feature map too, should we just handle that via AttributeAccessInterface as well?  Seems better than having to get it right in the attribute metadata and make sure the two values are the same....  This applies to all of these clusters.

Reply:
That is correct. I have added `FeatureMap` to the list for all the derived clusters.